app/src/androidTest/java/org/mozilla/fenix/ui/SettingsPrivacyTest.kt

```diff
@@ -705,6 +705,90 @@ class SettingsPrivacyTest {
             }
         }
     }
+
+    @SmokeTest
+    @Test
+    fun verifyCookieBannerReductionTest() {
+        val webSite = "voetbal24.be"
```

Comment:
Could checking with a real site make the test flaky? As we don't have control over the site, could we have a local site instead?

Reply:
Please don't feel the need to block on this comment, maybe this is something that could be improve in the feature.